### DISCUSSION TITLE
Stop with timer

### DISCUSSION THREAD

**Discussion by dkdue** [Ch4r0ne/ARK-Ascended-Server-Manager#2] [Ideas]
Sorry for my English, but I use a translator, I have several ideas:
-Put an auto-updater with timer*
-Put a Stop button with a timer*
-In the timer* you can put a message that works with rcon and the players can read it

*5 - 10 - 15 minute timer as desired by the administrator

I really like your application, congratulations

**Comment by Ch4r0ne** [comment]
Hello @dkdue 

Thank you very much for your feedback and suggestions! We appreciate your interest in our application and your support. Your ideas are great and would definitely contribute to an improved user experience. We will include them in our roadmap and explore how we can implement them in future updates.
